e2e-tests/uat.env

```diff
@@ -12,4 +12,4 @@ CARD_HOLDER_NAME_XPAY=Test Test
 CARD_NUMBER_VPOS=5255000010002856
 CARD_EXPIRATION_DATE_VPOS=1230
 CARD_CCV_VPOS=123
-CARD_HOLDER_NAME_VPOS=Test Test
\ No newline at end of file
+CARD_HOLDER_NAME_VPOS=Test Test
```

Comment:
The payment method variable id is missing

Reply:
Sorry, added to both the env files